### ISSUE TITLE
Support ARM builds for Linux

### ISSUE THREAD

**Issue by Webreaper** [Webreaper/Damselfly#313]
Raspberry pi....

**Comment by akashkj** [comment]
Came for this and saw it already there. Need for ARM64

```
WARNING: The requested image's platform (linux/amd64) does not match the detected host platform (linux/arm64/v8) and no specific platform was requested
81a1522a95794881d54a49b17c4e117553c02d6f0cf8769cb7d9411bbdd83815

```

**Comment by Webreaper** [comment]
See also #295

**Comment by bburkett52** [comment]
Anny update on ARM support (for RPi4)?
Thanks,
~Bob

**Comment by Webreaper** [comment]
It's already there and working, both in Docker and as a standard install. Have you actually tried it?

**Comment by bburkett52** [comment]
Yes I did try it. I had it installed on an AMD Ubuntu system successfully. When I tried to move to an ARM RPi4 Raspberry OS, I get the error indicating it wrong processor.

**Comment by Webreaper** [comment]
Oh, my apologies..... I didn't read this ticket properly. I thought it was for my other project.

I don't know if I can support RPi4 - as there's some x64 dependencies which may not work, I think. But I'll try and have a look when I get a chance.

**Comment by Keegan-Wright** [comment]
Hiya, any updates regarding raspberry pi support?

**Comment by Webreaper** [comment]
Sorry, no. As mentioned above, it's likely I won't be able to support RPi, but I haven't had a chance to even investigate.
